tests/platform_tests/mellanox/test_psu_power_threshold.py

```diff
@@ -0,0 +1,292 @@
+import logging
+import pytest
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.mellanox_data import get_platform_data
+from tests.common.utilities import wait_until
+from tests.platform_tests.thermal_control_test_helper import mocker_factory
+from mellanox_thermal_control_test_helper import MockerHelper, PsuPowerThresholdMocker
+
+pytestmark = [
+    pytest.mark.asic('mellanox'),
+    pytest.mark.topology('any')
+]
+
+logger = logging.getLogger(__name__)
+
+mocker = None
+
+MAX_PSUS = None
+
+@pytest.fixture
+# We can not set it as module because mocker_factory is function scope
+def mock_power_threshold(request, duthosts, rand_one_dut_hostname, mocker_factory):
+    global mocker
+    global MAX_PSUS
+
+    psudaemon_restarted = False
+
+    duthost = duthosts[rand_one_dut_hostname]
+    platform_data = get_platform_data(duthost)
+    MAX_PSUS = platform_data['psus']['number']
+
+    all_psus_supporting_thresholds = True
+    try:
+        for psu_index in range(MAX_PSUS):
+            _ = mocker.read_psu_power_threshold(psu_index + 1)
```

Comment:
"_=", no need?

Reply:
this is not the latest commit. we need the output to log in case there is an error, eg. not all necessary files exist.
please refer to the latest commit.